corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -76,7 +78,8 @@ def _last_submission_needs_update(last_submission, received_on_datetime, build_v
     if not (last_submission and last_submission.submission_date):
         return True
     time_difference = received_on_datetime - last_submission.submission_date
-    update_frequency = timedelta(seconds=60 * 15) if debounce else timedelta(seconds=0)
+    debounce_delay = settings.USER_REPORTING_METADATA_UPDATE_FREQUENCY
+    update_frequency = timedelta(seconds=debounce_delay) if debounce else timedelta(seconds=0)
```

Comment:
I don't think that this will never be less than a minute. Any reason to not have it defined in minutes instead?

Reply:
no I guess not

Reply:
23ef14a